sdk/hybridcompute/Azure.ResourceManager.HybridCompute/src/autorest.md

```diff
@@ -59,4 +59,7 @@ rename-rules:
   URI: Uri
   Etag: ETag|etag
 
+models-to-treat-empty-string-as-null:
```

Comment:
consider to add tests for this change

Reply:
We can't cover this with SDK tests, because we can't set [AgentConfiguration](https://github.com/Azure/azure-sdk-for-net/blob/d4302e6ffe5c1b2a6b7a7348f94dc320d50531ef/sdk/hybridcompute/Azure.ResourceManager.HybridCompute/src/Generated/Models/MachineProperties.cs#L85) with .NET SDK.